pkg/ui/src/views/shared/components/summaryBar/index.tsx

```diff
@@ -163,7 +164,16 @@ export class SummaryHeadlineStat extends React.Component<SummaryHeadlineStatProp
   render() {
     return <div className="summary-headline">
       <div className="summary-headline__value">{computeValue(this.props.value, this.props.format)}</div>
-      <div className="summary-headline__title">{this.props.title}</div>
+      <div className="summary-headline__title">
+        {this.props.title}
+        <div className="section-heading__tooltip">
+          <ToolTipWrapper text={this.props.tooltip}>
```

Comment:
Is it just me or is this a weird way for the markup of this component to appear?  I would think the info icon should be automatic, and the only config required would be the contents of the tooltip, which would then be the children?  Something to think about when you refactor this.

Reply:
Yeah, agree with your proposed usage in the other issue.